test/jit/test_freezing.py

```diff
@@ -1927,6 +1927,30 @@ def test_pool3d_batchnorm(self):
                 FileCheck().check("aten::to_dense").check_next("return").run(mod.graph)
                 self.assertTrue(torch.allclose(sub_model(inp), mod(inp)))
 
+    @unittest.skipIf(not torch._C.has_mkldnn, "MKL-DNN build is disabled")
+    @skipIfNoTorchVision
+    def test_layernorm(self):
+        with set_default_dtype(torch.float):
+            model = torchvision.models.resnet18()
+            N, C, H, W, = 10, 3, 224, 224
+            for param in ((model.conv1, [W // 2], torch.randn(N, C, H, W)),
+                          (model.conv1, [H // 2, W // 2], torch.randn(N, C, H, W)),
+                          (torch.nn.Linear(H, W), [W], torch.randn(N, C, W)),):
+
+                for layernorm in (torch.nn.LayerNorm(param[1]),
+                                  torch.nn.LayerNorm(param[1], elementwise_affine=False)):
+                    sub_model = torch.nn.Sequential(param[0], layernorm)
+                    sub_model.eval()
+                    mod = torch.jit.freeze(torch.jit.script(sub_model))
+                    self.run_pass("convert_frozen_ops_to_mkldnn", mod.graph)
+                    # if weight and bias are present and shape is the last dimension
+                    # we should convert `aten::layer_norm` to `prim::MKLDNNLayerNorm`
+                    if layernorm.elementwise_affine and len(param[1]) == 1:
+                        FileCheck().check("prim::MKLDNNLayerNorm").check_count("aten::to_dense", 1, exactly=True).run(mod.graph)
+                    else:
+                        FileCheck().check_count("aten::to_dense", 1, exactly=True).run(mod.graph)
```

Comment:
i think i commented this on other pr but this isnt actually checking anything, since we expect to_dense in both if branches. you should check if aten::layernorm exists